src/Components/Reviews.jsx

```diff
@@ -1,12 +1,14 @@
 import { ReviewCard } from "./ReviewCard";
 
-export const Reviews = ({ reviews }) => {
+export const Reviews = ({ reviews, isLoading }) => {
   return (
     <main>
       <header>
         <h2>Reviews</h2>
       </header>
-      <ReviewCard reviews={reviews} />
+      <section>
+        <ReviewCard reviews={reviews} isLoading={isLoading} />
```

Comment:
Feel ReviewCard could be confusingly named when it represents multiple reviews opposed to a single one